local-cli/templates/HelloWorld/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,18 +1,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.helloworld"
-    android:versionCode="1"
-    android:versionName="1.0">
+    package="com.helloworld">
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
 
-    <uses-sdk
-        android:minSdkVersion="16"
-        android:targetSdkVersion="22" />
-
     <application
       android:name=".MainApplication"
-      android:allowBackup="true"
```

Comment:
According to https://stackoverflow.com/questions/12648373/what-is-androidallowbackup this will cause a lint warning if not set. I think we could just set it to false by default.

Reply:
@janicduplessis fixed.

Reply:
Wouldn't it be beneficial to most app users to have automatic backups? Developers who aren't android savvy will not know to change this setting; and malicious user's can root their phone and get the data regardless. At the very least, there should be RN documentation about it.

https://developer.android.com/guide/topics/data/autobackup.html#EnablingAutoBackup